.github/workflows/website-preview.yml

```diff
@@ -2,6 +2,7 @@ name: Preview website on fork
 
 on:
   push:
+  workflow_dispatch:
     
```

Comment:
@harshulgupt I don't think we need to change this. It just provides an option to run the workflow manually.

Reply:
I see!

Reply:
Just a note- the website preview workflow runs every time we make changes.